### PR TITLE
Add responsive styling

### PR DESCRIPTION
**Pull request by devonharless**
* Add new fav icons
* Add js for mobile menu
* Add mobile nav styles
* Adjust font sizes for mobile
* Update illustrations

### CODE REVIEW

**Comment by devonharless** [comment]
mobile

File: source/assets/javascripts/application.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+    e.preventDefault();
+
+    $(".nav-links").toggleClass("active");
+    $(".nav-links.active a").on("click", dismissMobileMenu);
+  }
+
+  function dismissMobileMenu(e) {
+    $(".nav-links").removeClass("active");
+  }
+});
```
Comment (adarsh) [inline source/assets/javascripts/application.js:14]: Would it be possible to extract this to something like `javascripts/mobile_menu.js`?

File: source/assets/stylesheets/_navigation.scss [hunk 2]
```diff
@@ -6,6 +6,15 @@
   }
 }
 
+.menu-trigger {
+  display: none;
+  text-align: right;
+
+  @media (max-width: 768px) {
```
Comment (adarsh) [inline source/assets/stylesheets/_navigation.scss:13]: I wonder if this `768px` width is used frequently enough to be extracted to a constant?

**Review by adarsh — APPROVED** [review]
👍

**Comment by adarsh** [comment]
We can also put the Code for America logo up here. There is an incoming quote/face also 🦊

File: source/assets/javascripts/application.js (hunk 1 above)
Comment (devonharless) [reply source/assets/javascripts/application.js:14]: yep!

File: source/assets/stylesheets/_navigation.scss (hunk 2 above)
Comment (devonharless) [reply source/assets/stylesheets/_navigation.scss:13]: yep!

**Pull request merged**